gpicsync.py

```diff
@@ -215,6 +215,8 @@ def getFileList(dir):
     for fileName in sorted(os.listdir ( dir )):
         if (fnmatch.fnmatch ( fileName, '*.JPG' )
          or fnmatch.fnmatch ( fileName, '*.jpg' )
+         or (fnmatch.fnmatch ( fileName, '*.TIF' )
```

Comment:
Oops copy paste fail there. Please add the extra ).

On Sun, Feb 22, 2015 at 1:18 PM, Oleksandr Skrypnyk <
notifications@github.com> wrote:

> In gpicsync.py
> <https://github.com/notfrancois/GPicSync/pull/2#discussion_r25128489>:
>
> > @@ -215,6 +215,8 @@ def getFileList(dir):
> >      for fileName in sorted(os.listdir ( dir )):
> >          if (fnmatch.fnmatch ( fileName, '*.JPG' )
> >           or fnmatch.fnmatch ( fileName, '*.jpg' )
> > +         or (fnmatch.fnmatch ( fileName, '*.TIF' )
>
> extra openning parentheses here which leads to exception
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/notfrancois/GPicSync/pull/2/files#r25128489>.
>